pkg/apis/servicecatalog/register.go

```diff
@@ -56,6 +56,13 @@ func addKnownTypes(scheme *runtime.Scheme) error {
 		&metav1.GetOptions{},
 
 		&Broker{},
```

Comment:
I also would like to know the excruciating details of these. The comments on the individual types are not helpful.